modules/managed-cluster-notification-policy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="managed-cluster-notification-policy_{context}"]
+= Cluster notification policy
+
+Cluster notifications are designed to keep you informed about the health of your cluster and high impact events that affect it.
+
+Most cluster notifications are generated and sent automatically, to ensure that you are immediately informed of problems or important changes to the state of your cluster.
+
+In certain situations, Red Hat Site Reliability Engineering (SRE) manually creates and sends cluster notifications to provide additional context and guidance for a complex issue.
```

Comment:
```suggestion
In certain situations, Red Hat Site Reliability Engineering (SRE) creates and sends cluster notifications to provide additional context and guidance for a complex issue.
```

Is it important to call out manual vs automated notifications?

Reply:
@jewzaam In theory it's useful to call out the extra care and service of SRE over automated systems, but otherwise not really.